packages/expo-file-system/src/FileSystem.types.ts

```diff
@@ -1,16 +1,36 @@
+export enum FileSystemSessionType {
```

Comment:
Could you describe options of this enum? Like what `background` means and what's the drawback of using it.